# Description: Add missing AUTOKEYBOARD gconf option indexes,
#   otherwise the enum values do not match the indexes in gvm_settings
# Upstream: http://bugzilla.gnome.org/show_bug.cgi?id=525345
--- gnome-volume-manager-2.22.1/src/manager.c	2008-03-31 16:30:50.000000000 +0200
+++ gnome-volume-manager-2.22.1.new/src/manager.c	2008-03-31 16:31:13.000000000 +0200
@@ -121,6 +121,8 @@
 	AUTOBURN_DATA_CD_COMMAND,
 	AUTOIPOD,
 	AUTOIPOD_COMMAND,
+	AUTOKEYBOARD,
+	AUTOKEYBOARD_COMMAND,
 	AUTOMOUNT_DRIVES,
 	AUTOMOUNT_MEDIA,
 	AUTOOPEN,
